packages/web-components/README.md

```diff
@@ -72,6 +72,14 @@ Add the following into the top level CSS file for your project.
 @import "@ukic/web-components/dist/core/core.css";
 ```
 
+In order to be rendered consistently across browsers and in line with modern standards, each of the ICDS components uses styles from a global CSS file based on [Normalize.css](https://necolas.github.io/normalize.css/). 
```

Comment:
```suggestion
In order to be rendered consistently across browsers and in line with modern standards, each of the ICDS components use styles from a global CSS file based on [Normalize.css](https://necolas.github.io/normalize.css/). 
```

Reply:
Just had a look online about this and I think the singular form is correct? E.g. [this article](https://www.writersdigest.com/write-better-fiction/does-each-use-a-plural-or-singular-verb-grammar-rules#:~:text=or%20Singular%20Verb%3F-,Before,-we%20can%20figure)